alpha/alpha-server/pom.xml

```diff
@@ -201,6 +201,11 @@
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-all</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>2.1.1</version>
```

Comment:
please add this in the dependency managements and use the property like ${kafka.version}

Reply:
Thank you for reminding me，I have just fixed it